proposals/interface-types/working-notes/scenarios/fetch.md

```diff
@@ -0,0 +1,222 @@
+# Fetch Url
```

Comment:
Same comment here as with `directory.md`: it'd be good to add a big bold disclaimer saying this is speculative and ahead of a writeup in working-notes and Explainer.md.